client/src/components/Details.js

```diff
@@ -32,7 +38,7 @@ export class Details extends React.Component {
         aria-labelledby={label_id}
       >
         <span aria-hidden className="IBDetails__TogglerIcon">
-          { isOpen ? "▼" : "►" }
+          { isOpen ? opened_drawer_icon || "▼" : closed_drawer_icon || "►" }
```

Comment:
Alternatively, could use 
```
Details.defaultProps = {
  opened_drawer_icon: "▼",
  closed_drawer_icon: "►",
}
```

Don't actually feel too strongly one way or the other, but I think we (myself included) probably under-use default props.